scripts/setup.sh

```diff
@@ -317,9 +317,15 @@ wget -q $FUNCTION_REPOSITORY_RELEASE_URL -O $FUNCTION_ZIP_PACKAGE
 echo "- extracting archive [$FUNCTION_ZIP_PACKAGE]"
 mkdir -p $GCP_FUNCTION_NAME
 unzip -o -q ./$FUNCTION_ZIP_PACKAGE -d ./$GCP_FUNCTION_NAME || exit
-echo "- deploy the function [$GCP_FUNCTION_NAME]"
 pushd ./$GCP_FUNCTION_NAME || exit
 GCP_FUNCTION_TIMEOUT=$(( QUERY_INTERVAL_MIN*60 ))
+
+if [[ $(gcloud functions list --filter=$GCP_FUNCTION_NAME --project="$GCP_PROJECT" --format="value(name)")  ]]; then
+  echo "- updating the function [$GCP_FUNCTION_NAME]"
```

Comment:
I think this is fine, but we should warn user (and ask for confirmation) that any manual modification of function env variables will be overwritten

Reply:
I function is already deployed we should skip interactive prompts for function size etc...

Reply:
Right now I've changed it in a way that during an update besides the question about the project all other questions are omitted.  Also now no Env variables will be updated as it was in update.sh. Not sure if it's ok, (I'm just worried and looking in the future, how we'll be delivering additional environmental variables in case of additional features), nevertheless it's consistent with the previous approach.